src/views/Login.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import svg from "../assets/mobile_login.svg";
+import googleIcon from "../assets/google_icon.svg";
+import facebookIcon from "../assets/facebook_icon.svg";
+import barefootLogo from "../assets/barefoot_logo.svg";
+import Input from "../components/Input";
+
+function LoginForm(props) {
+  const googleAuth = () => {
+    window.open(
+      "https://bravo-bfn-be.herokuapp.com/api/v1/user/auth/google/",
+      "_self"
+    );
+  };
+  const facebookAuth = () => {
+    window.open(
+      "https://bravo-bfn-be.herokuapp.com/api/v1/user/auth/facebook/",
```

Comment:
also those Url i would be more easier if you use link from env

Reply:
thought about that, thanks.